src/components/Form/Form.tsx

```diff
@@ -1,110 +1,161 @@
-            </div>
-            <div className={styles.wrapper}>
-                <label className={styles.label}>
-                    Release Date
-                    <input
-                        value={values.date}
-                        onChange={e => setValues('date', e.target.value)}
-                        type='date'
-                        placeholder='Select Date'
-                        className={styles.input}
-                        required
-                    />
-                </label>
-            </div>
-            <div className={styles.wrapper}>
-                <label className={styles.label}>
-                    Movie URL
-                    <input
-                        value={values.urlMovie}
-                        onChange={e => setValues('urlMovie', e.target.value)}
-                        type='text'
-                        placeholder='Movie URL here'
-                        className={styles.input}
-                        required
-                    />
-                </label>
-            </div>
-            <div className={styles.wrapper}>
-                <label className={styles.label}>
-                    Genre
-                    <MultiSelect onChange={values => setValues('genre', values)} initialValues={values.genre}/>
-                </label>
-            </div>
-            <div className={styles.wrapper}>
-                <label className={styles.label}>
-                    Overview
-                    <input
-                        value={values.overview}
-                        onChange={e => setValues('overview', e.target.value)}
-                        type='text'
-                        placeholder='Overview here'
-                        className={styles.input}
-                        required
+            if(!values.release_date) {
+                error.release_date = 'Date is required'
+            }
+
+            if(!values.poster_path) {
+                error.poster_path = 'Movie url is required'
+            }
+
+            if(!values.genres?.length) {
+                error.genres = 'Select at least one genre is proceed'
+            }
+
+            if(!values.overview) {
+                error.overview = 'Overview is required'
+            }
+
+            if(!values.runtime) {
+                error.runtime = 'Runtime is required'
+            }
+
+            return error
+        }
+    });
+
+    return(
+        <form onSubmit={formik.handleSubmit}>
+            <FormikProvider value={formik}>
+                <p className={styles.label}>{titleID}</p>
+                <p className={styles.movieID}>{movieID}</p>
+                <InputField
+                    name='title'
+                    label='Title'
+                    placeholder='Title'
+                    onChange={formik.handleChange}
+                    value={formik.values.title}
+                    error={formik.errors.title}
+                    type='text'
+                />
```

Comment:
You have a lot of pretty similar inputs. Maybe you can simplify them? IDK the best wat. Maybe create new component?
```js
const FormInput = ({name, label, placeholder, type}) => (
<InputField 
  name={name}
  label={label}
  placeholder={placeholder}
  onChange={formik.handleChange}
  value={formik.values[name]}
  error={formik.errors[name]}
  type={type}
/>
)

...

<FormInput name='title' label='Title' placeholder='Title' type='text'  />
```